dbx/about-compatibility.rst

```diff
@@ -0,0 +1,57 @@
+About Compatibility Tables
+--------------------------
+
+We include compatibility tables for each version of the driver to guide
+your decisions on what versions you need to ensure your environment
+remains fully operational.
+
+We maintain the following tables for each driver:
+
+- :ref:`MongoDB compatibility <mongodb-compatibility-tables-{+driver+}>`
+- :ref:`Language compatibility <language-compatibility-tables-{+driver+}>`
+
+Read the sections below for detailed explanations of each table.
+
+.. _mongodb-compatibility-tables-{+driver+}:
+
+MongoDB Compatibility Tables
```

Comment:
I think this might be better in the singular form unless we plan to feature multiple MongoDB compatibility tables per driver. 

Applies to all instances (including targets).

Reply:
Sounds good.